netci.groovy

```diff
@@ -50,6 +50,7 @@ def project = GithubProject
 
                         skipTestsWhenResultsNotFound = false
                     }
+					Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
```

Comment:
Also update the `windowsScript` definition above to use dev15.

Or leave these changes out of this PR and send them as a followup PR (to `master`, since that's the current source of truth for our CI definitions).